app/routes/check-answers.js

```diff
@@ -32,12 +32,12 @@ module.exports = {
           actions: { items: [{ href: '/claim/vet-visit-date', text: 'Change', visuallyHiddenText: 'change date of visit' }] }
         },
         {
-          key: { text: 'Vet name' },
+          key: { text: 'Vet\'s name' },
```

Comment:
Which ticket covers this content change?

Reply:
Highlighted in yellow in all "species" tickets e.g. https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/169090

Hmmm but not in the prototype :(